concepticondata/concepticon.tsv

```diff
@@ -1694,7 +1694,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1693	THURSDAY	Time	The fourth day of the week in Europe and in systems using the ISO 8601 norm; the fifth day of the week in the United States of America.	Person/Thing	
 1694	WEDNESDAY	Time	The third day of the week in Europe and in systems using the ISO 8601 norm; the fourth day of the week in the United States of America.	Person/Thing	
 1695	TUESDAY	Time	The second day of the week in Europe and in systems using the ISO 8601 norm; the third day of the week in the United States of America.	Person/Thing	
-1696	SURRENDER	Warfare and hunting	The act of surrendering to the enemy.	Action/Process	
+1696	SURRENDER	Warfare and hunting	To give up to the enemy.	Action/Process	
```

Comment:
At least in English we have the related meaning of "to yield", e.g., "to surrend the village/the arms". Should we include it? Is it found in the concept lists and in the data?

Reply:
Borin's definition includes both meanings, Buck distinguishes it from "yield", the others are without explanation. I think my definition could be understood to include both meanings.

Reply:
Yes, if it is not in any list, let's keep it out. The elicitations, in any case, probably pick the definition you have. :+1: